src/components/Articles.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../utils/utils";
+import ArticleCard from "./ArticleCard";
+
+const Articles = () => {
+  const [articles, SetArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((data) => {
+      SetArticles(data);
+    });
```

Comment:
Please add an isLoading state so that people know that data is still being fetched.